src/StopHorizon.h

```diff
@@ -76,7 +76,7 @@ namespace agent_model {
         void init(double s) {
 
             _sActual = s;
-            _elements.clear();
+            if (_elements.size() > 0) _elements.clear();
```

Comment:
does the command fail if element size is == 0 ? Or why did you introduce this line?